SoundSwitch/Framework/NotificationManager/Notification/NotificationBanner.cs

```diff
@@ -20,23 +20,48 @@
 using SoundSwitch.Common.Framework.Audio.Device;
 using SoundSwitch.Framework.Audio;
 using SoundSwitch.Framework.Banner;
+using SoundSwitch.Framework.Banner.Position;
 using SoundSwitch.Framework.NotificationManager.Notification.Configuration;
 using SoundSwitch.Localization;
+using SoundSwitch.Model;
 using SoundSwitch.Properties;
 
 namespace SoundSwitch.Framework.NotificationManager.Notification
 {
-    public class NotificationBanner : INotification
+    internal class NotificationBanner : INotification
     {
         public NotificationTypeEnum TypeEnum => NotificationTypeEnum.BannerNotification;
         public string Label => SettingsStrings.notificationOptionBanner;
 
         public INotificationConfiguration Configuration { get; set; }
-
         private readonly BannerManager _bannerManager = new();
         private readonly BannerPositionFactory _bannerPositionFactory = new();
 
-        public bool SupportIcon => true;
+        private IPosition BannerPosition() => _bannerPositionFactory.Get(Configuration.BannerPosition);
+
+        public void NotifyDefaultChanged(DeviceFullInfo audioDevice)
+        {
+            var toastData = new BannerData
+            {
+                Image = audioDevice.LargeIcon.ToBitmap(),
+                Text = audioDevice.NameClean,
+                Position = BannerPosition()
```

Comment:
Then this get changed to this
```suggestion
                Position = BannerPosition
```